src/index.html

```diff
@@ -11,7 +11,142 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"    method="post">
+      <fieldset class="group">
+        <legend>Personal information</legend>
+        <label class="form__field">
+          Surname:
+          <input
+          type="text"
+          name="Surname"
+          autocomplete="off"
+          >
+        </label>
+        <label class="form__field">
+          Name:
+          <input
+          type="text"
+          name="Name"
+          autocomplete="off"
+          >
+        </label>
+        <label class="form__field">
+          How old are you?
+          <input
+          type="number"
+          name="How old are You?"
```

Comment:
[checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) 4th item, use camelCase for `name` attribute, fix it EVERYWHERE
```suggestion
          name="howOldAreYou"
```